powermon/__init__.py

```diff
@@ -196,95 +189,45 @@ def main():
     #     protocol: PI30MAX
     port_config = device_config["port"].copy()
     log.debug(f"portconfig: {port_config}")
-    port = get_port(port_config)
+    portType = port_config["type"]
+    portPath = port_config["path"]
+    portBaud = port_config["baud"]
```

Comment:
not all ports follow this config
this where i started and then ran into trouble with ever increasing parameters. this is why i have moved to passing the config object, so each port can have its own parameters